opm/output/eclipse/RestartIO.cpp

```diff
@@ -223,24 +223,23 @@ namespace {
         rstFile.write("ZGRP", groupData.getZGroup());
     }
 
-    void writeGroupLGR([[maybe_unused]] int                           sim_step,
-                       [[maybe_unused]] const UnitSystem&             units,
-                       [[maybe_unused]] const Schedule&               schedule,
-                       [[maybe_unused]] const Opm::SummaryState&      sumState,
-                       [[maybe_unused]] const std::vector<int>&       ih,
-                       [[maybe_unused]] EclIO::OutputStream::Restart& rstFile,
-                       [[maybe_unused]] const std::string&            lgr_tag)
+    void writeGroupLGR(int                           sim_step,
+                       const UnitSystem&             units,
+                       const Schedule&               schedule,
+                       const Opm::SummaryState&      sumState,
+                       const std::vector<int>&       ih,
+                       EclIO::OutputStream::Restart& rstFile,
+                       const std::string&            lgr_tag)
     {
-        // temporary disabled until captureDeclaredGroupDataLGR and the respective tests are fixed
-        // const size_t simStep = static_cast<size_t> (sim_step);
-        // auto  groupData = Helpers::AggregateGroupData(ih);
+        const size_t simStep = static_cast<size_t> (sim_step);
```

Comment:
Could you use [`std::size_t`](https://en.cppreference.com/w/cpp/types/size_t.html) (from `<cstddef>`) instead of the untagged `size_t` type here, please?